en/simulation/hitl.md

```diff
@@ -1,101 +1,133 @@
 # Hardware in the Loop Simulation \(HITL\)
 
-Hardware in the loop simulation is a simulation mode where the autopilot is connected to the simulator and all flight code runs on the autopilot. This approach has the benefit of testing the actual flight code on the real processor.
+Hardware-in-the-Loop (HITL) is a simulation mode in which normal PX4 firmware is run on real flight controller hardware. This approach has the benefit of testing most of the actual flight code on the real hardware.
 
-## Step 1: Enable HITL mode and configure airframe
+PX4 supports HITL for multicopters (using jMAVSim or Gazebo) and fixed wing (using X-Plane demo/full version or Gazebo). 
 
-Connect the autopilot directly to QGroundControl.
 
-First enable HITL mode in the safety setup as shown in the screenshot below. Select the "Enabled" in the drop down on the HITL enabled section \(first on the page\):![QGroundControl HITL configuration](../../assets/qgc_hitl_config.png)
+## HITL-Compatible Airframes {#compatible_airframe}
 
-PX4 supports HITL for multicopters \(using jMAVSim or Gazebo\) and fixed wing \(using X-Plane demo or full version or Gazebo\). Generally any compatible airframe can be put into HITL mode. Compatible airframes are right now:
+The current list of compatible airframes vs Simulators is:
```

Comment:
@dagar 
1. What makes a simulator _compatible_? I.e. what change would we have to make to another frame to ensure it is supported?
2. Or conversely, if I look at configuration, how can I tell that something is properly HITL enabled?

Reply:
Most of the barrier is the mixer and how it interacts with the simulation. After that it's a matter of getting matching models on each side.

Reply:
As I understand it the mixer translates movement commands from controllers into airframe-specific motor commands. So what I think you are saying is that:
- the mixer for a particular airframe might not match actual behaviour in the simulated vehicle. 
- what you have to do therefore is update the model in Gazebo so that it matches the output of the mixer.

Is that right? 

I don't understand "After that it's a matter of getting matching models on each side." so I intepreted that as update gazebo. However you could be suggesting that PX4 also needs to be modified - perhaps with a HIL mode for the mixer?